pcsx2/CDVD/CDVDcommon.cpp

```diff
@@ -303,6 +304,21 @@ static CDVD_SourceType m_CurrentSourceType = CDVD_SourceType::NoDisc;
 
 void CDVDsys_SetFile(CDVD_SourceType srctype, std::string newfile)
 {
+#ifdef WIN32
+	if (Path::IsAbsolute(newfile))
+	{
+		const auto& splitPath = Path::SplitNativePath(newfile);
```

Comment:
Ah, missed this one - that's returning a new vector, so it shouldn't be stored as a reference (`const auto splitPath` would be fine)... surprised the compiler didn't whinge about it.